### ISSUE TITLE
Copy paste keyboard UX issue

### ISSUE THREAD

**Issue by Keyikedalube** [synfig/synfig#1713]
**Synfig version & platform**:
Building from source
Debian sid

**Issue description**:
1. Draw a rectangle & change its color to red
2. Try to copy the hex color value using **Ctrl+C**
3. Draw a circle and bring up its color properties dialog box
4. Paste **Ctrl+V**

Another rectangle layer gets pasted. I think this is due to bad input handling design.
When a user highlights the color code and hits Ctrl+C, the color should be copied because user's focus is on the dialog box now not the canvas workspace.

*Right-clicking copy and paste works but that's tedious. Keyboard saves a lot more time than mouse*

**Comment by Keyikedalube** [comment]
Looks like `Widget_ColorEdit` itself is the issue. Maybe this widget should be a dialog too?

Here's my test:
1. Try opening canvas properties dialog
2. Copy width value
3. And paste it to height

Copy-paste works fine

**Comment by rodolforg** [comment]
Thank you, @Keyikedalube !!
I was pretty sure that was a 'external' dialog problem (color, gradient, ...), that main app catch keypress events before those dialogs for accels!

So I checked the differences between Dialog_Color and CanvasProperties, and here it is:
`	add_accel_group(App::ui_manager()->get_accel_group());  `
This line makes in those problematic dialogs is the responsible to ensure the Actions' accel taking precedence regular actions.

Three solutions:
1. Just remove it. Consequence: when the dialog window is on focus, accels/shortcuts to application/main window actions don't work.
2. Remove it and implement `on_key_press_event()` for all problematic dialogs in a similar way I did in MainWindow when I fixed #257 and 'we' discovered this problem you reported.
3. Similar above, but create a base dialog class with the `on_key_press_event()` implemented, and problematic dialogs inherits from it.
